core/eth/utils.go

```diff
@@ -15,6 +15,9 @@ import (
 
 var (
 	maxNumberOfQuorums = 192
+
+	// ErrPaymentDoesNotExist is returned when a reservation/deposit does not exist (is zero)
+	ErrPaymentDoesNotExist = errors.New("payment does not exist")
```

Comment:
Is this specific to reservations?

Reply:
no, also ondemand. do you want them separate?